src/Stryker.Core/Stryker.Core/Reporters/Progress/ProgressBar.cs

```diff
@@ -28,7 +28,7 @@ public class ProgressBar : IProgressBar, IDisposable
 
         public void Start(int maxTicks, string message) => _progressBar = new ShellProgressBar.ProgressBar(Math.Max(maxTicks, 1), message, _options);
 
-        public void Tick(string message) => _progressBar.Tick(message);
+        public void Tick(string message) => _progressBar?.Tick(message);
 
```

Comment:
So you've basically fixed [this](https://www.youtube.com/watch?v=1M02bAWDFkI) :D